package.json

```diff
@@ -3,6 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-preset-airbnb": "^2.4.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
It's not clear to me here if we need stage-2 or stage-0

Reply:
I was able to make it run using only `babel-preset-airbnb`.

Reply:
:thinking: I'm super sure I tried that. I must have imagined it, because you are right.
I removed the stages, thanks!